exercises/structs/structs3.rs

```diff
@@ -57,6 +57,15 @@ mod tests {
         assert!(package.is_international());
     }
 
+    #[test]
+    fn create_local_package() {
+        let sender_country = String::from("Canada");
+        let recipient_country = sender_country.clone();
+
+        let package = Package::new(sender_country, recipient_country, 1200);
+
+        assert!(!package.is_international());
+    }
```

Comment:
Sorry to be a bother, but could you add a space here, before the next test?

Reply:
Absolutely! Not sure how I missed that.